src/models/User.js

```diff
@@ -72,10 +76,11 @@ class User extends Resource {
 
   /**
    * @param {ChangePasswordRequest} changePasswordRequest
+   * @param {object} queryParameters
    * @returns {Promise<UserCredentials>}
    */
-  changePassword(changePasswordRequest) {
-    return this.client.changePassword(this.id, changePasswordRequest);
+  changePassword(changePasswordRequest, queryParameters) {
+    return this.client.changePassword(this.id, changePasswordRequest, queryParameters);
```

Comment:
same question, breaking changes?

Reply:
No issues with adding a parameter, they can just be omitted